src/app/psbt.ts

```diff
@@ -331,6 +338,167 @@ const Psbt = function (config) {
             return psbtModule.broadcastPsbt(psbt);
         },
 
+        signMultipleUtxosPsbtForSend: async ({
+            address, pubKey, selectedUtxos,
+            ownedUtxos, destinationBtcAddress, sendFeeRate
+        }) => {
+            const utxosWithInscription = selectedUtxos.filter(utxo => utxo.inscriptionId);
+            const utxosWithoutInscription = selectedUtxos.filter(utxo => !utxo.inscriptionId);
+            if (utxosWithInscription.length === 0 && utxosWithoutInscription.length === 0) {
+                throw new Error('At least one ordinal or utxo is required.');
+            }
+            const provider = SessionStorage.get(SessionsStorageKeys.DOMAIN);
+            if (provider !== 'alby') {
+                throw new Error('Signing not supported.');
+            }
+
+            const spendableUtxos = ownedUtxos.filter(utxo => !selectedUtxos.some(ownedUtxo => ownedUtxo.txid === utxo.txid))
+                .filter((x) => x.status.confirmed)
+                .filter((x) => x.value > 10000)
+                .filter(utxo => !utxo.inscriptionId)
+                .sort((a, b) => b.value - a.value);
+
+            const selectedAmountWithoutInscription = utxosWithoutInscription.reduce((acc, utxo) => acc + utxo.value, 0);
+            let cardinalAmount = selectedAmountWithoutInscription;
+            let calculatedFee = 0;
+
+            const inputs = [...utxosWithInscription, ...utxosWithoutInscription];
+            let isCardinalAdded = false;
+
+            do {
+                let shouldAddChangeOutput = utxosWithInscription.length > 0 || isCardinalAdded;
+                calculatedFee = cryptoModule.calculateFee({
```

Comment:
I think because of the way we calculate the fee, we don't need all the signatures before the confirm step